README.md

```diff
@@ -15,3 +15,40 @@ To run the site locally with Docker, use the following command:
 ```bash
 docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll jekyll serve
 ```
+
+
+
+#### Side note for those on non-linux machines
+ 
```

Comment:
Oh reading the PR again, why doesn't docker for mac work on a Mac? I don't mind minikube, but I see that as a very useful thing for folks on Windows, but not on Mac, since the docker for Mac thing works pretty smoothly. 

But if your workflow is just minikube, I am okay with this. Do please fix the text to state that minikube on OSX is one option (standard docker for mac is another).